src/ekf/ekf.cpp

```diff
@@ -151,24 +177,84 @@ void mocapFilter::updateMocap(geometry_msgs::TransformStamped msg)
   tf::transformMsgToTF(msg.transform, measurement);
   tf::Matrix3x3(measurement.getRotation()).getRPY(roll, pitch, yaw);
   Eigen::Matrix<double, 6, 1> y;
-  y << measurement.getOrigin().getX(), // NWU to NED
```

Comment:
We may need to add a parameter about which frame mocap is in.  In the simulator, Odometry is in NED.  However, the actual motion capture system publishes in NWU.  (which is confusing.  We may just want to convert the motion capture output to NED and save us all some headache.)  I think that for now, I'll leave this, but we just need to remember that Mocap is backwards until we fix it.  I'll create an issue that we can solve later.